testsuite/shared/src/main/java/org/jboss/as/test/integration/management/util/CustomCLIExecutor.java

```diff
@@ -62,8 +62,9 @@
             + MANAGEMENT_HTTPS_PORT;
 
     private static Logger LOGGER = Logger.getLogger(CustomCLIExecutor.class);
-    private static final int CLI_PROC_TIMEOUT = 5000;
-    private static final int STATUS_CHECK_INTERVAL = 2000;
+
+    private static final int CLI_PROC_TIMEOUT = TimeoutUtil.adjust(30000);
+    private static final int STATUS_CHECK_INTERVAL = TimeoutUtil.adjust(2000);
```

Comment:
Apologies; my IDE search was inaccurate. The "failureResponse" response is used in one test.